### PR TITLE
Cherry picked config changes (Converted static links to config)

### PR DESCRIPTION
**Pull request by bhasin85**

### CODE REVIEW

File: grails-app/views/wps/show.gsp [hunk 1]
```diff
@@ -8,8 +8,8 @@
     <body>
         <div class="portalheader">
             <div class="container">
-                <a class="btn" role="button" href="https://imos.aodn.org.au/imos123/home">
-                    <img src="https://static.emii.org.au/images/logo/IMOS-Ocean-Portal-logo.png" alt="IMOS logo">
+                <a class="btn" role="button" href="${grailsApplication.config.portal.imos_home}">
+                    <img src="${grailsApplication.config.portal.imos_logo}" alt="IMOS logo">
```
Comment (pmbohm) [inline grails-app/views/wps/show.gsp]: The alt tag needs to be generic

File: grails-app/conf/Config.groovy [hunk 2]
```diff
@@ -240,10 +226,26 @@ baselayers = [
...
-    siteHeader = "[not configured]"
+    siteHeader = "Open Access to Ocean Data"
     motdUrl = "https://static.emii.org.au/motd"
-
-    logo = "images/favicon.ico"
+    logo = "https://static.emii.org.au/images/logo/AODN_logo_fullText.png"
+    secondaryLogo = "https://static.emii.org.au/images/logo/IMOS_logo-stacked-reversed-Converted.png"
+    icon = "https://static.emii.org.au/images/aodn/favicon.ico"
+
+    imos = "https://imos.aodn.org.au"
```
Comment (pmbohm) [inline grails-app/conf/Config.groovy]: This is config for configs sake.
There is only a few items that should be added to config
1) site wide email address
1) condition_of_use

File: grails-app/conf/Config.groovy [hunk 3]
```diff
@@ -256,11 +258,11 @@ portal {
...
+        <a title="Email us for help in using this site" target="_blank" class="external" href="mailto:${portal.aodn_email}?subject=Portal enquiry - ">Contact</a> <b>|</b>
         <a title="Data usage acknowledgement" target="_blank" class="external" href="${help.url}user-guide-introduction/aodn-portal/data-use-acknowledgement">Acknowledgement</a> <b>|</b>
         <a title="Disclaimer information" target="_blank" class="external" href="${help.url}user-guide-introduction/aodn-portal/disclaimer">Disclaimer</a> <b>|</b>
-        <a title="Australian Ocean Data Network" target="_blank" class="external" href="http://imos.org.au/aodn.html">AODN</a> <b>|</b>
-        <a title="Integrated Marine Observing System" target="_blank" class="external" href="http://www.imos.org.au">IMOS</a> <b>|</b>
+        <a title="Australian Ocean Data Network" target="_blank" class="external" href="${portal.imos}/aodn.html">AODN</a> <b>|</b>
+        <a title="Integrated Marine Observing System" target="_blank" class="external" href="${portal.imos}">IMOS</a> <b>|</b>
```
Comment (pmbohm) [inline grails-app/conf/Config.groovy]: pointless
`title="Australian Ocean Data Network"`

File: grails-app/views/wps/jobComplete.gsp [hunk 4]
```diff
@@ -9,5 +9,5 @@ The eMII Team.
 
 
 --
-If you require assistance with this service please contact us at info@aodn.org.au. The job reference is ${uuid}.
+If you require assistance with this service please contact us at ${grailsApplication.config.portal.aodn_email}. The job reference is ${uuid}.
```
Comment (pmbohm) [inline grails-app/views/wps/jobComplete.gsp]: Should be generic:

`${grailsApplication.config.portal.siteEmail}`

File: grails-app/conf/Config.groovy [hunk 5]
```diff
@@ -141,14 +141,6 @@ environments {
         // Set to true if you want to test interaction with new servers. This turns
         // your portal instance into an open proxy and can be dangerous.
```
Comment (pmbohm) [inline grails-app/conf/Config.groovy]: Your commit message will have no meaning in terms of the git log

File: grails-app/conf/Config.groovy [hunk 6]
```diff
@@ -256,11 +248,11 @@ portal {
...
+        <a title="Email us for help in using this site" target="_blank" class="external" href="mailto:${portal.contactEmail}?subject=Portal enquiry - ">Contact</a> <b>|</b>
         <a title="Data usage acknowledgement" target="_blank" class="external" href="${help.url}user-guide-introduction/aodn-portal/data-use-acknowledgement">Acknowledgement</a> <b>|</b>
         <a title="Disclaimer information" target="_blank" class="external" href="${help.url}user-guide-introduction/aodn-portal/disclaimer">Disclaimer</a> <b>|</b>
-        <a title="Australian Ocean Data Network" target="_blank" class="external" href="http://imos.org.au/aodn.html">AODN</a> <b>|</b>
-        <a title="Integrated Marine Observing System" target="_blank" class="external" href="http://www.imos.org.au">IMOS</a> <b>|</b>
+        <a title="Australian Ocean Data Network" target="_blank" class="external" href="https://imos.aodn.org.au/aodn.html">AODN</a> <b>|</b>
+        <a title="Integrated Marine Observing System" target="_blank" class="external" href="https://imos.aodn.org.au">IMOS</a> <b>|</b>
```
Comment (pmbohm) [inline grails-app/conf/Config.groovy]: These are not the right URL's

File: grails-app/conf/Config.groovy [hunk 7]
```diff
@@ -240,10 +226,16 @@ baselayers = [
...
 portal {
-    siteHeader = "[not configured]"
+    siteHeader = "Open Access to Ocean Data"
     motdUrl = "https://static.emii.org.au/motd"
+    logo = "https://static.emii.org.au/images/logo/AODN_logo_fullText.png"
+    secondaryLogo = "https://static.emii.org.au/images/logo/IMOS_logo-stacked-reversed-Converted.png"
+    icon = "https://static.emii.org.au/images/aodn/favicon.ico"
+
+    contactEmail = "info@aodn.org.au"
+    imosConditionOfUse = "http://imos.org.au/imostermsofuse0.html"
```
Comment (pmbohm) [inline grails-app/conf/Config.groovy]: Still says `imos`

**Comment by pmbohm** [comment]
@bhasin85 Just a couple of little issues

File: grails-app/conf/Config.groovy [hunk 8]
```diff
@@ -256,11 +248,11 @@ portal {
 
     footerContent =
         """
-        <a title="Email us for help in using this site" target="_blank" class="external" href="mailto:info@aodn.org.au?subject=Portal enquiry - ">Contact</a> <b>|</b>
+        <a title="Email us for help in using this site" target="_blank" class="external" href="mailto:${portal.contactEmail}?subject=Portal enquiry - ">Contact</a> <b>|</b>
         <a title="Data usage acknowledgement" target="_blank" class="external" href="${help.url}user-guide-introduction/aodn-portal/data-use-acknowledgement">Acknowledgement</a> <b>|</b>
         <a title="Disclaimer information" target="_blank" class="external" href="${help.url}user-guide-introduction/aodn-portal/disclaimer">Disclaimer</a> <b>|</b>
         <a title="Australian Ocean Data Network" target="_blank" class="external" href="http://imos.org.au/aodn.html">AODN</a> <b>|</b>
-        <a title="Integrated Marine Observing System" target="_blank" class="external" href="http://www.imos.org.au">IMOS</a> <b>|</b>
+        <a title="Integrated Marine Observing System" target="_blank" class="external" href="https://imos.aodn.org.au">IMOS</a> <b>|</b>
```
Comment (pmbohm) [inline grails-app/conf/Config.groovy]: This is not right! Its a link to the IMOS home page
`http://www.imos.org.au/`

File: grails-app/views/wps/show.gsp [hunk 9]
```diff
@@ -40,10 +40,10 @@
...
                            All feedback is very welcome. For help and information about this site
-                           please contact <a href="mailto:info@aodn.org.au">info@aodn.org.au</a></p>
+                           please contact <a href="mailto:${grailsApplication.config.portal.contactEmail}">${grailsApplication.config.portal.contactEmail}</a></p>
                     </div>
                     <div class="col-md-8">
-                        <p>Use of this web site and information available from it is subject to our <a href="http://imos.org.au/imostermsofuse0.html">
+                        <p>Use of this web site and information available from it is subject to our <a href="${grailsApplication.config.portal.conditionOfUse}">
                            Conditions of use
                         </a></p>
                         <p>© 2016 IMOS</p>
```
Comment (pmbohm) [inline grails-app/views/wps/show.gsp]: Sorry I didnt spot this. Maybe remove the whole `<p>© 2016 IMOS</p>`
We dont even have copyright on the landing page
@bhasin85

Comment (bhasin85) [inline grails-app/views/wps/show.gsp]: @pmbohm  Does it look better :+1:

**Pull request merged**